File: 04_sorting/04_02_bubble_sort/bubble_sort.js
const bubbleSort = (arr = []) => {
    let flag = true;
    for (let index = 0; index < arr.length - 1 && flag; index++) {
        flag = false;
        for (let j_index = 0; j_index < arr.length - index - 1; j_index++) {
            if (arr[j_index] > arr[j_index + 1]) {
                [arr[j_index], arr[j_index + 1]] = [arr[j_index + 1], arr[j_index]];
                flag = true;
            }
        }
    }
}

const array = [13, 46, 24, 52, 20, 9]

console.log("Before:-");
console.log(array);
bubbleSort(array)
console.log("After:-");
console.log(array);

File: 04_sorting/04_02_bubble_sort/bubble_sort.py
def bubbleSort(arr: list):
    for i in range(len(arr)-1):
        for j in range(len(arr)-i-1):
            if arr[j] > arr[j+1]:
                arr[j], arr[j+1] = arr[j+1], arr[j]
                
elements = [13, 46, 24, 52, 20, 9]
print("Before:-")
print(elements)
bubbleSort(elements)
print("After:-")
print(elements)